packages/amplify-provider-awscloudformation/src/system-config-manager.js

```diff
@@ -286,7 +286,22 @@ function getProfileCredentials(profileName) {
       }
     });
   }
-  return normalizeKeys(profileCredentials);
+  profileCredentials = normalizeKeys(profileCredentials);
+  validateCredentials(profileCredentials, profileName);
+  return profileCredentials;
+}
+
+function validateCredentials(credentials, profileName) {
+  if (credentials && credentials.accessKeyId === undefined) {
+    const err = new Error(`Profile configuration is invalid for: ${profileName}, missing aws_access_key_id`);
+    logger('validateCredentials', [profileName])(err);
+    throw err;
```

Comment:
Add `err.stack = undefined;` so we'll not print stacktrace and that's a better DX for customers.